bin/hsw-cli

```diff
@@ -592,93 +592,76 @@ class CLI {
-        this.log('  $ balance: Get wallet balance.');
-        this.log('  $ history: View TX history.');
-        this.log('  $ pending: View pending TXs.');
-        this.log('  $ coins: View wallet coins.');
-        this.log('  $ account list: List account names.');
+        this.log('  $ abandon [hash]: Abandon a transaction.');
         this.log('  $ account create [account-name]: Create account.');
         this.log('  $ account get [account-name]: Get account details.');
+        this.log('  $ account list: List account names.');
         this.log('  $ address: Derive new address.');
+        this.log('  $ balance: Get wallet balance.');
+        this.log('  $ block [height]: View wallet block.');
+        this.log('  $ blocks: List wallet blocks.');
         this.log('  $ change: Derive new change address.');
+        this.log('  $ coins: View wallet coins.');
+        this.log('  $ dump [address]: Get wallet key WIF by address.');
+        this.log('  $ get: View wallet.');
+        this.log('  $ history: View TX history.');
+        this.log('  $ import [wif|hex]: Import private or public key.');
+        this.log('  $ key [address]: Get wallet key by address.');
+        this.log('  $ listen: Listen for events.');
+        this.log('  $ lock: Lock wallet.');
+        this.log('  $ master: View wallet master key.');
+        this.log('  $ mktx [address] [value]: Create transaction.');
+        this.log('  $ pending: View pending TXs.');
+        this.log('  $ rescan [height]: Rescan for transactions.');
         this.log('  $ retoken: Create new api key.');
         this.log('  $ send [address] [value]: Send transaction.');
-        this.log('  $ mktx [address] [value]: Create transaction.');
+        this.log('  $ shared add [xpubkey]: Add key to wallet.');
+        this.log('  $ shared remove [xpubkey]: Remove key from wallet.');
         this.log('  $ sign [tx-hex]: Sign transaction.');
-        this.log('  $ zap [age?]: Zap pending wallet TXs.');
-        this.log('  $ abandon [hash]: Abandon a transaction.');
         this.log('  $ tx [hash]: View transaction details.');
-        this.log('  $ blocks: List wallet blocks.');
-        this.log('  $ block [height]: View wallet block.');
+        this.log('  $ unlock [passphrase] [timeout?]: Unlock wallet.');
         this.log('  $ view [tx-hex]: Parse and view transaction.');
-        this.log('  $ import [wif|hex]: Import private or public key.');
         this.log('  $ watch [address]: Import an address.');
-        this.log('  $ key [address]: Get wallet key by address.');
-        this.log('  $ dump [address]: Get wallet key WIF by address.');
-        this.log('  $ lock: Lock wallet.');
-        this.log('  $ unlock [passphrase] [timeout?]: Unlock wallet.');
+        this.log('  $ zap [age?]: Zap pending wallet TXs.');
+        this.log('');
+        this.log('Admin commands, they require api-token if node is run with wallet-wallet-auth flag.');
```

Comment:
Actually it requires `admin-token` and I think we can say here it is the `wallet-auth` flag, even though I know it may be `wallet-wallet-auth` -- perhaps link to the config guide? Like how you linked to API-docs below?

There is another layer which I don't know how to include here which is if wallet-auth is true, then each individual wallet requires its own wallet-token, for ANY command (the admin token overrides all individual wallet tokens). See https://hsd-dev.org/api-docs/#wallet-authentication

Reply:
Actually, the `token` is used for both: admin authentication and wallet authentication. It's just if the `token === adminToken` then you are admin, otherwise you have to provide wallet specific `token`. So in both cases `token` is used.

There's no such thing as `api-token` or `admin-token` in parameters (admin-token only exists in wallet config file)

Reply:
Added.
I also think it's authorization, not authentication (sorry).